qemu/tests/cfg/virtio_scsi_cdrom.cfg

```diff
@@ -0,0 +1,93 @@
+- virtio_scsi_cdrom:
+    virt_test_type = qemu
+    type = unattended_install
+    backup_image = yes
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
+    shutdown_cleanly = yes
+    shutdown_cleanly_timeout = 120
+    force_create_image = yes
+    guest_port_unattended_install = 12323
+    kernel = vmlinuz
+    initrd = initrd.img
+    # Throw errors if guest screen is inactive
+    inactivity_watcher = error
+    # Inactivity treshold to error the test
+    inactivity_treshold = 1800
+    image_verify_bootable = no
+    # Backup images from nfs when install failed
+    image_copy_on_error = no
+    # don't have enough RAM error from anaconda, please enlarge this value.
+    lowest_mem = 512
+    only x86_64
+    only virtio_scsi, virtio_blk
+    virtio_drive_letter = 'D'
+    variants:
+        - with_installation:
+            cd_format_cd1 = scsi-cd
+            i440fx:
+                cd_format_unattended = scsi-cd
+                Windows:
+                    cd_format_unattended = ide
```

Comment:
cd_format = scsi-cd
        - with_installation:
            i440fx:
                Windows:
                    cd_format_unattended = ide
                q35:
                    cd_format_unattended = ahci

Reply:
Thanks for your comment, have updated.